ppp/iteration_2_24102022.md

```diff
@@ -0,0 +1,23 @@
+### Plans 📆
+
+- Reiterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
```

Comment:
🔧 I associate "reiterating" with saying something multiple times, although it's not wrong according to dictionaries. 

```suggestion
- Iterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
```